admin_manual/configuration_files/federated_cloud_sharing_configuration.rst

```diff
@@ -21,18 +27,71 @@ servers.`` Leaving the checkboxes blank disables federated cloud sharing.
 3. In the Sharing section, check ``Allow users to share via link`` and ``Allow 
 users to send mail notification for shared files``.
```

Comment:
Ah, but it is required for the `Creating Federated Cloud Shares via Email` way you use later on. However, since this one is not enabled by default while all others are, I'd say move this to the section below.